build/template-build-and-prep-automation.yaml

```diff
@@ -129,11 +129,11 @@ steps:
         $AppsToCheck = @('MSIX\WPF\bin\Release\', 
                          'MSIX\WPF.Package\bin\AnyCPU\Release\WPF\', 
                          'NetCoreWinFormsWam\bin\Release\net6.0-windows10.0.17763.0\', 
-                         'NetCoreWinFormsWam\bin\Release\netcoreapp3.1\', 
```

Comment:
so we do not need this (netcoreapp3.1) anymore?

Reply:
They have been removed